Source/Scene/Material.js

```diff
@@ -435,6 +434,17 @@ Material.prototype.update = function (context) {
     uniformId = loadedImage.id;
     var image = loadedImage.image;
 
+    // Images transcoded from KTX2 can contain multiple mip levels:
+    // https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_texture_basisu
+    var mipLevels;
+    if (Array.isArray(image)) {
+      // highest detail mip should be level 0
+      mipLevels = image.slice(1, image.length).map(function (mipLevel) {
```

Comment:
Throughout, use `arraySlice` instead of `.slice`

Reply:
In fact, from a performance standpoint, we wouldn't actually use `map` and `slice` here at all.

Reply:
This area shouldn't be a performance problem since it only gets called once when the texture is first loaded